server/src/main/webapp/WEB-INF/rails/webpack/views/pages/new-environments.tsx

```diff
@@ -22,9 +22,11 @@ import {EnvironmentsAPIs} from "models/new-environments/environments_apis";
 import {FlashMessage, MessageType} from "views/components/flash_message";
 import {EnvironmentsWidget} from "views/pages/new-environments/environments_widget";
 import {Page, PageState} from "views/pages/page";
+import {DeleteConfirmModal} from "../components/modal/delete_confirm_modal";
+import {DeleteOperation, SaveOperation} from "./page_operations";
 
-interface State {
-  onSuccessfulSave: (msg: m.Children) => void;
+interface State extends SaveOperation, DeleteOperation<EnvironmentWithOrigin> {
+  // onSuccessfulSave: (msg: m.Children) => void;
```

Comment:
Can we remove this commented line as it is not required?

Reply:
sure, will do.